setup.sh

```diff
@@ -150,54 +173,75 @@ else
                 fi
         ;;
         -i) echo "Initialize terraform dir: ${ENV}"
-                if [ $2 ] ; then
-                        init $2
-                else
-                        for folder in ${COMPONENTS[@]}
-                        do
-                                init $folder
-                        done
+                does_stack_config_exist
+                if [ $? = 1 ] ; then
```

Comment:
if you flip the `0` and `1` senses (see comment above in definition of `does_stack_config_exist`), then I think you can use it directly in the `if` block:

```sh
if does_stack_config_exist; then
```

Reply:
Tested and works a treat 👍